helm/aladdin-demo/values/values.PROD.yaml

```diff
@@ -0,0 +1,4 @@
+# Values here will overwrite values.yaml defaults. These values are used when deploying to PROD cluster
+
+elasticsearch:
+  create: false
```

Comment:
we may want to have a teamwide discussion on this - Jeff thinks it's best for the default file to have prod values

Reply:
In that case we'd have a LOCAL.values instead of PROD.values?

Reply:
most projects have both - one other route we talked about in the past was having the default values file just list all the keys, and only fill the values that are constant between all environments. Then each values.x.yaml file overrides all values that vary between environments (Even if it's the same in say, 3 out 4 environments) - this was to kind of address an issue we've faced in the past where we thought we were overriding something but we weren't.

Reply:
Okay. If we have time we can have a quick discussion during sprint planning today.